specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -1664,9 +1765,95 @@
           "description": "The managed application artifact type."
         }
       },
+      "required": [
+        "name",
+        "type",
+        "uri"
+      ],
       "description": "Managed application artifact."
     },
-    "ApplicationProviderAuthorization": {
+    "ApplicationNotificationPolicy": {
+      "properties": {
+        "notificationEndpoints": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationNotificationEndpoint"
+          },
+          "description": "The managed application notification endpoint."
+        }
+      },
+      "required": [
+        "notificationEndpoints"
+      ],
+      "description": "Managed application notification policy."
+    },
+    "ApplicationNotificationEndpoint": {
+      "properties": {
+        "uri": {
+          "type": "string",
+          "description": "The managed application notification endpoint uri."
+        }
+      },
+      "required": [
+        "uri"
+      ],
+      "description": "Managed application notification endpoint."
+    },
+    "ApplicationPackageLockingPolicyDefinition": {
+      "properties": {
+        "allowedActions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The deny assignment excluded actions."
+        }
+      },
+      "description": "Managed application locking policy."
+    },
+    "ApplicationDeploymentPolicy": {
+      "properties": {
+        "deploymentMode": {
+          "$ref": "#/definitions/DeploymentMode",
+          "description": "The managed application deployment mode."
+        }
+      },
+      "required": [
+        "deploymentMode"
+      ],
+      "description": "Managed application deployment policy."
+    },
+    "ApplicationManagementPolicy": {
+      "properties": {
+        "mode": {
+          "$ref": "#/definitions/ApplicationManagementMode",
+          "description": "The managed application management mode."
+        }
+      },
+      "description": "Managed application management policy."
+    },
+    "ApplicationPolicy": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The policy name"
+        },
+        "policyDefinitionId": {
+          "type": "string",
+          "description": "The policy definition Id."
+        },
+        "parameters": {
+          "type": "string",
+          "description": "The policy parameters."
+        },
+        "sku": {
```

Comment:
SKU is a deprecated concept in policy.  You shouldn't have any reason to need this